yarn-project/end-to-end/src/uniswap_trade_on_l1_from_l2.test.ts

```diff
@@ -333,30 +327,7 @@ describe('uniswap_trade_on_l1_from_l2', () => {
 
     // 4.1 Owner approves user to swap on their behalf:
     const nonceForSwap = new Fr(3n);
-    const swapMessageHash = await hashPayload([
-      sponsorAddress.toField(),
-      uniswapL2Contract.address.toField(),
-      FunctionSelector.fromSignature(
-        'swap_public((Field),(Field),Field,(Field),Field,Field,Field,(Field),Field,Field,(Field),(Field),Field)',
-      ).toField(),
-      ownerAddress.toField(),
-      wethCrossChainHarness.l2Bridge.address.toField(),
-      new Fr(wethAmountToBridge),
-      daiCrossChainHarness.l2Bridge.address.toField(),
-      nonceForWETHTransferApproval,
-      new Fr(uniswapFeeTier),
-      new Fr(minimumOutputAmount),
-      ownerAddress.toField(),
-      secretHashForDepositingSwappedDai,
-      new Fr(deadlineForDepositingSwappedDai),
-      ownerEthAddress.toField(),
-      ownerEthAddress.toField(),
-      nonceForSwap,
-    ]);
-    await ownerWallet.setPublicAuth(swapMessageHash, true).send().wait();
-
-    // 4.2 Call swap_public from user2 on behalf of owner
-    const withdrawReceipt = await uniswapL2Contract
+    const action = uniswapL2Contract
```

Comment:
rename action to functionCall?

Reply:
We have something that is a `FunctionCall`, the return value from the `request` xD so think that might be more confusing.

Reply:
You guys discuss the names of local variables in tests..? Wow, that's a thorough review indeed.